go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/libp2p/go-libp2p-pubsub
 
 require (
 	github.com/gogo/protobuf v1.3.1
-	github.com/hashicorp/golang-lru v0.5.4
+	github.com/hashicorp/golang-lru v0.5.4 // indirect
```

Comment:
Is this an artifact from your local branch or do we need to change the dependency to indirect anyway?